packages/material-ui-envinfo/package.json

```diff
@@ -15,9 +15,15 @@
   "bugs": {
     "url": "https://github.com/mui-org/material-ui/issues"
   },
+  "homepage": "https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-envinfo",
   "repository": {
     "type": "git",
     "url": "https://github.com/mui-org/material-ui",
     "directory": "packages/material-ui-envinfo"
+  },
+  "files": [],
+  "scripts": {
+    "build": "tar xfz $(npm pack) && mv package build",
```

Comment:
Need to investigate windows support. Might need a dedicated script.

Not using `scripts/copy-files` since that doesn't fit at all (includes changelog - too big, mutates manifest).

Reply:
works now on windows.